baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ValidateConstantMessage.java

```diff
@@ -45,6 +46,19 @@
     private static final Matcher<ExpressionTree> VALIDATE_METHODS =
             MethodMatchers.staticMethod()
                     .onClassAny("org.apache.commons.lang3.Validate", "org.apache.commons.lang.Validate");
+    private static final Matcher<ExpressionTree> VALIDATE_METHODS_MESSAGE_THIRD_PARAM =
+            Matchers.anyOf(
+                    MethodMatchers.staticMethod()
+                            // all org.apache.commons.lang.Validate methods have the message as the second arg
```

Comment:
actually there's one, `allElementsOfType`, where the message is the third arg... updated matcher